### PR TITLE
Correcting statement about Service ports

### LINKED ISSUE #13034: Issue with k8s.io/docs/concepts/services-networking/service/
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**

There is a discrepancy in the documentation regarding multi-port support of Service Object. In the ""Defining a Service" section, it was mentioned that: "At the moment, you can only set a single port and protocol for a Service". However, in the later section of "Multi-Port Services", it shows Kubernetes supports multiple port definitions on a Service object.

**Proposed Solution:**

Delete the sentence of in the "At the moment, you can only set a single port and protocol for a Service" in the ""Defining a Service" section

**Page to Update:**
https://kubernetes.io/docs/concepts/services-networking/service/

 1.13 Documentation about Service

### PR DESCRIPTION
**Pull request by rajeshdeshpande02**
Correcting details about service ports and protocol. Removed statement saying Service supports single port definition only and added details about multiport support.

This also fixes #13034

### CODE REVIEW

**Comment by DanyC97** [comment]
@sftim i remember you done some work on https://github.com/kubernetes/website/pull/12669 , are you able to comment from tech side ? 

I haven't dived into the code ...

**Comment by sftim** [comment]
We already document this: https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -84,8 +84,7 @@ number that pods expose in the next version of your backend software, without
 breaking clients.
 
 `TCP` is the default protocol for services, and you can also use any other
-[supported protocol](#protocol-support). At the moment, you can only set a
-single `port` and `protocol` for a Service.
+[supported protocol](#protocol-support). As many `Services` need to expose more than one port, Kubernetes supports multiple port definitions on a `Service` object. Each port definition can have the same or a different `protocol`.
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md]: To follow the style guide, “Services” should be initial caps but not in backticks. “protocol” in backticks seems fine to me: it's talking about how you configure the Service object, not the `service` section of some YAML.

Comment (sftim) [reply content/en/docs/concepts/services-networking/service.md]: Consider wrapping this line to match the rest of the file.

**Comment by sftim** [comment]
Thanks for this PR @rajeshdeshpande02

**Comment by sftim** [comment]
I'm happy to see the misleading documentation fixed. I think we could tweak the formatting though.

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (rajeshdeshpande02) [reply content/en/docs/concepts/services-networking/service.md]: @sftim Thanks for your review. Fixed formatting issues as per your suggestions. Please have a look and let me know if you think more changes required

**Comment by sftim** [comment]
This looks OK. The rest of the document isn't 100% following the style guide; however, fixing that ought to come via a different PR.
/lgtm

**Comment by rajeshdeshpande02** [comment]
@DanyC97 @sftim Thank you for your review. As it looks good to you, can you please approve this PR

**Pull request merged**